cloud/scaling.rst

```diff
@@ -0,0 +1,42 @@
+Scaling
+#######
+
+Citus Cloud provides self-service scaling to deal with increased load. The web interface makes it easy to either add new worker nodes or increase existing nodes' memory capacity.
+
+For most cases either approach to scaling is fine and will improve performance. However there are times you may want to choose one over the other. If the cluster is reaching its disk limit then adding more nodes is the best choice. Alternately, if there is still a lot of headroom on disk but performance is suffering, then scaling node RAM is the way to go.
+
+Both adjustments are available in the formation configuration panel of the settings tab:
+
+.. image:: ../images/cloud-formation-configuration.png
+
+Clicking the node change link provides two sliders:
+
+.. image:: ../images/cloud-nodes-slider.png
+
+The first slider, **count**, scales out the cluster by adding new nodes. The second, **RAM**, scales it up by changing the amount of RAM on existing nodes.
+
+After you adjust the sliders and accept the changes, Citus Cloud begins applying the changes subject to the user-specified maintenance window:
+
+.. image:: ../images/cloud-maintenance-window.png
+
+Scaling Up (increasing node size)
+---------------------------------
+
+Resizing node RAM works by creating a PostgreSQL follower for each node, where the followers are provisioned with the desired amount of RAM. It takes an average of forty minutes per hundred gigabytes of data for the primary nodes' data to be fully synchronized on the followers. After the synchronization is complete, Citus Cloud does a quick failover from the existing primary nodes to their followers which takes about one minute. The creation and switchover process uses the same well-tested replication mechanism that powers Cloud's high-availability feature.
```

Comment:
Could we show example screen-shots of an increase of the second slider here? (i.e. only bumping that up), and maybe the first slider below? I'm not sure if it adds to the clarity, but was wondering if they would.